tests/e2e.rs

```diff
@@ -9,8 +9,8 @@ use testcontainers::{clients, core::WaitFor};
 mod common;
 
 const POLICIES: &[&str] = &[
-    // SHA: 59e34f482b40
-    "registry://ghcr.io/kubewarden/tests/pod-privileged:v0.1.9",
+    // SHA: 01690a10f9c3
+    "registry://ghcr.io/kubewarden/tests/pod-privileged:v0.2.5",
```

Comment:
oh yes, that was the old policy written with AssemblyScript that cannot be ran anymore.